### PR TITLE
Scaffold front page

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: css/site.styl [hunk 1]
```diff
@@ -329,6 +351,25 @@ img
 
 // ******************************************************
 
+// HEADER
+
+header#header
```
Comment (varffs) [inline css/site.styl]: can just be `#header`

File: css/site.styl [hunk 2]
```diff
@@ -329,6 +351,25 @@ img
...
+  border-top: 0
+
+// FRONT PAGE 
+
+#front-splash
+  padding: 0
+
+.splash-image
```
Comment (varffs) [inline css/site.styl]: if a unique element can have id selector

File: footer.php [hunk 3]
```diff
@@ -55,10 +59,10 @@
         <?php if (!empty($mailchimp_url)) { ?>
 
           <div class="col col-l col-l-4">
-            <p>Mailing List</p>
+            <h5 class="margin-bottom-micro">Mailing List</h5>
             <form method="post" id="mailchimp-form" name="mailchimp-form">
-              <input type="email" size="30" value="email" name="email">
-              <input type="submit" value="Subscribe" name="subscribe" class="button">
+              <input type="email" size="30" placeholder="email" value="" name="email">
```
Comment (varffs) [inline footer.php:64]: how come this doesnt at least add the correct form from the site option?

Comment (mikeyrayvon) [inline footer.php:64]: Because I assume were doing ajax

Comment (mikeyrayvon) [inline footer.php:64]: And just allowing some random markup to be dropped in via meta field could lead to some fuckery no?

Comment (varffs) [inline footer.php:64]: i just mean the action url for the form element could be a non ajax fallback. not a full form embed in the site admin [and i think siteadmin html additions are fine tho cus admin responsibility to keep WP secure

File: header.php [hunk 4]
```diff
@@ -112,23 +115,24 @@
...
-            ?>
-        </div>
+        
+        <?php  
+          if (!empty($apply_url) && !empty($apply_end) && ( time() <= $apply_end ) && !$publish_exhibitors ) { 
+            echo '<a href="' . $apply_url . '" class="col col-l col-l-2 flex-col justify-center align-center button">';
+            echo __('[:en]Apply[:es]Apply');
+            echo '</a>';
+          } else {
+            echo '<div class="col col-l col-l-2"></div>';
```
Comment (varffs) [inline header.php]: what is `col-1-2`?

File: lib/meta-boxes.php [hunk 5]
```diff
@@ -71,9 +71,30 @@ function igv_cmb_metaboxes() {
...
-    'name'     => __( 'Address', 'cmb2' ),
+    'name'     => __( 'City (English)', 'cmb2' ),
     'desc'     => __( '', 'cmb2' ),
-    'id'       => $prefix . 'exhibitor_address',
+    'id'       => $prefix . 'exhibitor_city_en',
+    'type'     => 'text',
+  ) );
+
+  $exhibitor_metabox->add_field( array(
+    'name'     => __( 'City (Español)', 'cmb2' ),
```
Comment (varffs) [inline lib/meta-boxes.php:81]: normal metaboxes can be localized with the CMB2 q-translate plugin which is probably easier than this way. [its just annoying site options cannot]

File: front-page.php [hunk 6]
```diff
@@ -0,0 +1,508 @@
...
+//
+// SPLASH
+//
+//
+
+if (!empty($splash_id)) { ?>
+  <section id="front-splash" class="section">
+    <?php 
+      echo (!empty($splash_link) ? '<a href="' . esc_url( $splash_link ) . '">' : ''); 
+      echo wp_get_attachment_image($splash_id, 'splash', null, array('class' => 'splash-image'));
```
Comment (varffs) [inline front-page.php]: I thought this was going to be a post or something [or maybe a gallery/carousel of posts]? Just choosing image and link seems a bit weird. We could just have a post search input to choose anything

**Comment by varffs** [comment]
Also the labels on the admin site options are a bit confusing. Perhaps it doesnt matter as we will have to do training cus its got some complex, but it took me a while to work it all out

**Comment by mikeyrayvon** [comment]
where could it be better?

File: lib/meta-boxes.php (hunk 5 above)
Comment (mikeyrayvon) [inline lib/meta-boxes.php:81]: this one says it works on theme options https://github.com/jmarceli/integration-cmb2-qtranslate

Comment (mikeyrayvon) [inline lib/meta-boxes.php:81]: I can try to migrate to that in another branch?

**Comment by varffs** [comment]
Front Page Headline and Description seem more like maybe splash heading and copy or something to me, but is only once worked out. Group on ticket info was a bit confusing, seem more like ticket type or class. Maybe it isnt really bad but it will need learning I think. Maybe was just because its not all on one Front Page section [but then again often used in other places?]

**Pull request merged**